tests/sparsezoo/utils/utilities.py

```diff
@@ -23,6 +23,14 @@
 from sparsezoo.utils import CACHE_DIR
```

Comment:
let's not include these changes right now, since they're not in the scope for testing `iter_batches`

Reply:
Resolved